docs/reference/manifest/extendedpermission.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ExtendedPermission element in the manifest file
+description: ''
+ms.date: 03/04/2020
+localization_priority: Normal
+---
+
+# `ExtendedPermission` element
+
+Defines an extended permission the add-in needs to work. The `ExtendedPermission` element is a child element of [ExtendedPermissions](extendedpermissions.md).
+
+> [!IMPORTANT]
+> This element is only available in the [Outlook add-ins preview requirement set](../objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) against Exchange Online. Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
```

Comment:
We should confirm this from Hitesh in IDC: "Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment." Was this a requirement?

Reply:
This is usually the case for preview API, etc - we strongly advise folks to use only released API in their production add-ins.